{
  "contract": "erc20_short_address",
  "tool": "mythril",
  "start": 1563372324.911674,
  "end": 1563372333.352881,
  "duration": 8.441206932067871,
  "analysis": {
    "error": null,
    "issues": [
      {
        "address": 160,
        "code": "assert(msg.data.length == size + 4)",
        "debug": "The exception is triggered under the following conditions:\n\ncalldatasize_NonPayloadAttackableToken: 0x4\ncalldata_NonPayloadAttackableToken_0: 0xa9059cbb00000000000000000000000000000000000000000000000000000000\ncallvalue: 0x0\n",
        "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ",
        "filename": "/dataset/short_addresses/erc20_short_address.sol",
        "function": "transfer(address,uint256)",
        "lineno": 10,
        "title": "Exception state",
        "type": "Informational"
      }
    ],
    "success": true
  }
}